crates/nu-cli/src/commands/to_json.rs

```diff
@@ -37,13 +44,13 @@ impl WholeStreamCommand for ToJSON {
         &[
             Example {
                 description:
-                    "Outputs an unformatted JSON string representing the contents of this table",
+                    "Outputs an unformatted json string representing the contents of this table",
```

Comment:
all caps `JSON` is technically correct. I wouldn't mind sticking with that.

Reply:
Sorry, I thought I saw the it being used in two different ways in the docs before I touched it and I just unified them to one format.  I'll make them all upper case.

Reply:
Fix is pushed and waiting on tests to complete